core/src/test/scala/unit/kafka/metrics/MetricsTest.scala

```diff
@@ -58,6 +58,8 @@ class MetricsTest extends KafkaServerTestHarness with Logging {
     // force creation not client's specific metrics.
     createAndShutdownStep("group0", "consumer0", "producer0")
 
+    assertNotNull(DelayedFetchMetrics)
```

Comment:
Add comment here: "this assertion is only used for creating the metrics for DelayedFetchMetrics, it should never fail, but should not be removed".